### ISSUE TITLE
Sound of engines is weird

### ISSUE THREAD

**Issue by speeder** [endless-sky/endless-sky#69]
I am playing on windows.

When the ship is thrusting (by me pressing the forward button, or the AI manuevering the ship), no sound happens, EXCEPT when the thrusting stop,then it make some hard to hear noise, like a faint click.

It becomes obvious when the AI is desperately trying to stop the ship to land and the ship is heading in a direction you can't turn your ship into (lack of sprite?), because the AI will turn the ship and flash the engines on and off several times in a second, it does stop the ship perfect (wonderfull AI!) but it makes a really weird noise (because the flashing engine on and off makes the mentioned faint click every time it turns off, so you hear it several times in a second)

**Comment by endless-sky** [comment]
The engine sound is a *very* soft hum, so it's possible that it would be playing and you wouldn't notice it. On Linux I don't hear a click when the sound stops (for any of the engine types) so that may be a Windows-specific issue. Do you hear a similar click at the end of other looping noises, like the laser beams firing?

The audio is the one part of the game that I haven't put any real work into, yet - the sounds that are there are just placeholders, to verify that the audio engine actually works.

**Comment by speeder** [comment]
I don't know if the lasers click, maybe they are too loud for that.

the engine I noticed because at first I thought the game had no sound at all, since it has no music, and no sound effects beside combat, one of the first thing you notice in any space game is the engine sound, it was very weird when I pressed the thrust, and no sound happened, it was too eeerily silent.

THEN I noticed the clicking, it happens when you (or the AI) let go the finger of the button, not when it is supposed to loop I believe, maybe it is getting freed before the player stopped playing. (also I hear no engine sound, and my speakers are really big)
